### PR TITLE
fix: added max-height to checkbox

### PR DESCRIPTION
**Pull request by i035249**

### CODE REVIEW

File: src/packages/QuestionCheckbox.vue [hunk 1]
```diff
@@ -2,37 +2,48 @@
...
                   :input-value="active"
                   :true-value="item.value"
+                  id="question-checkbox-v-list-item-action-checkbox"
                 ></v-checkbox>
               </v-list-item-action>
               <v-list-item-content>
-                <v-list-item-title v-text="item.name"></v-list-item-title>
+                <v-list-item-title 
+                  v-text="item.name">
+                  id="question-checkbox-v-list-item-title"
```
Comment (tomer-epstein) [inline src/packages/QuestionCheckbox.vue:45]: id should be inside the <v-list-item-title>

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sap-devx/inquirer-gui",
   "displayName": "Inquirer GUI",
-  "version": "0.2.0",
+  "version": "0.3.0",
```
Comment (tomer-epstein) [inline package.json:4]: 0.2.1

Comment (tomer-epstein) [reply package.json:4]: also the example app should be tested and consume version 0.2.1
https://github.com/SAP/inquirer-gui/blob/checkbox_add_max-height-slider/sample-app/package.json#L19

Comment (i035249) [reply package.json]: will be done in a separate PR as first we need to publish the inquirer-gui to npm before consuming it. no?

File: src/packages/QuestionCheckbox.vue (hunk 1 above)
Comment (i035249) [reply src/packages/QuestionCheckbox.vue]: done

**Review by slavik-lvovsky — APPROVED** [review]

**Pull request merged**